crates/macro/src/derive_props/field.rs

```diff
@@ -149,7 +148,7 @@ impl PropField {
                 _ => None,
             })?;
 
-        if meta_list.ident == "props" {
+        if meta_list.path.get_ident().map(|ident| ident.to_string()) == Some("props".to_owned()) {
```

Comment:
How about trying to avoid the `to_owned` allocation and use a helper fn like this:
```rust
fn match_path(path: &Path, id_str: &str) -> bool {
    if let Some(ident) = path.get_ident() {
        ident == id_str
    } else {
        false
    }
}
```

Reply:
I just found `Path::is_ident()`, which does that